packages/examples/src/ai.ts

```diff
@@ -19,7 +19,61 @@ const client = new MessariClient({
   apiKey: API_KEY,
 });
 
+// Get command line arguments
+const args = process.argv.slice(2);
+const useOpenAI = args.includes("openai");
+const useStreaming = args.includes("stream");
+
 async function main() {
+  if (useOpenAI) {
```

Comment:
imo its confusing for the end user to differentiate between `createChatCompletion` and `createChatCompletionOpenAI`. We should pick one and hide that complexity from them